clients/fides-js/src/lib/tcf.ts

```diff
@@ -119,37 +153,73 @@ export const generateTcString = async ({
 };
 
 /**
- * Call tcf() to configure Fides with tcf support (if tcf is enabled).
+ * Extract just the TC string from a FidesEvent. This will also remove parts of the
+ * TC string that we do not want to surface with our CMP API events, such as
+ * `vendors_disclosed` and our own AC string addition.
  */
-export const tcf = () => {
+const fidesEventToTcString = (event: FidesEvent) => {
```

Comment:
awesome, I think this means I'll have to make minimal changes, if any, on my TC str override work 🌟